hilti/toolchain/include/ast/operators/bitfield.h

```diff
@@ -75,4 +77,24 @@ right.
     }
 END_OPERATOR_CUSTOM_x
 
+BEGIN_OPERATOR_CUSTOM(bitfield, HasMember)
+    Type result(const hilti::node::Range<Expression>& /* ops */) const { return type::Bool(); }
+
+    bool isLhs() const { return false; }
+    auto priority() const { return hilti::operator_::Priority::Normal; }
+
+    const std::vector<Operand>& operands() const {
+        static std::vector<Operand> _operands =
+            {{{}, type::constant(type::Bitfield(type::Wildcard())), false, {}, "bitfield"},
+             {{}, type::Member(type::Wildcard()), false, {}, "<field>"}};
+        return _operands;
+    }
+
+    void validate(const expression::ResolvedOperator& i, operator_::position_t p) const {
+        detail::checkName(i.op0(), i.op1(), p.node);
+    }
+
+    std::string doc() const { return "Returns true if the bitfield's element has a value to provide."; }
```

Comment:
I fuzzed around with this a bit because "has a value" might seem a bit odd at first because in the standard use case (mapping an integer to a bitfield) all elements are always available anyways. But yeah,  "to provide" doesn't really help either.